app/utils/ServerRequests.tsx

```diff
@@ -108,4 +108,57 @@ export const searchLatestFiles = async(): Promise<FileCardType[] | void> => {
             return fileList;
         })
         .catch((error) => console.error(error));
-};  
\ No newline at end of file
+};  
+
+
+export const deleteItem = async (fileURL: string): Promise<boolean | void> => {
+    const requestHeaders = new Headers();
+    requestHeaders.append("Authorization", `Basic ${process.env.EXPO_PUBLIC_TOKEN}`);
+
+    const requestOptions = {
+        method: "DELETE",
+        headers: requestHeaders,
+        redirect: "follow"
+    };
+
+    fetch(machineURL+fileURL, requestOptions as RequestInit)
+        .then((response) => response.text())
+        .then((result) => {return true})
+        .catch((error) => {console.error(error); return false});
+}
+
+export const createFolder = async (folderName: string): Promise<boolean | void> => {
```

Comment:
If you mean that ``folderName`` should contain the entire url maybe rename it to ``folderURL``

Reply:
yep

Reply:
Should be fixed now, i renamed also the variable in delete, since we can delete folders as well with it